my-site.rb

```diff
@@ -2,7 +2,23 @@
 
 class MySite < Sinatra::Base
 
-  get "/" do
+  get "/index" do
+    erb :index
+  end
+
+  get "/projectsummary" do
```

Comment:
Sounds good. I can rename the files and then rename the routes using the underscore to separate multiple words when I start on wave 2.